frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetInstallSubmariner/InstallSubmarinerForm.tsx

```diff
@@ -915,6 +919,23 @@ export function InstallSubmarinerForm(props: { availableClusters: Cluster[] }) {
                                         setOpenStackInstanceTypes(copy)
                                     },
                                 },
+                                {
+                                    id: 'isAirGappedDeployment',
```

Comment:
Is it possible to run air-gapped deployment for managed OCP (ARO,ROSA) ? 

if it isn't you should make this field as hidden for managed services

Reply:
I have to check the general ACM behavior when it comes to Air-Gapped clusters - i.e. if they have a flag/config-param to convey that deployment is on air-Gapped clusters and if they restrict usage of that flag based on the cluster location. In general AFAICT, most of the air-gapped clusters are onPrem clusters and even if they use public clouds, the cluster VPC might be configured to block external access. Let me find more details about this. I want to handle this in a follow-up PR, hope its okay for you?

Reply:
Well, it seems that air-gapped/disconnected deployments are supported also for managed OCP, check [1].

Didn't find a central airgap/disconnected flag either, I can see that for ACM create-cluster (OSP or VMWARE) there is a disconnected section [2] that is copied into OCP's install-config **imageContentSources** field.


But yeah, sure, we can handle it in a follow-up PR if needed.

[1] 
https://github.com/redhat-cop/ocp-disconnected-docs/blob/main/AROInstall.md
[2]
![image](https://user-images.githubusercontent.com/29688326/204509164-a6429daa-24fe-42b1-9e67-c91358b5c7cd.png)

Reply:
thanks for checking @yboaron